qiskit/pulse/commands/instruction.py

```diff
@@ -45,7 +45,7 @@ def __init__(self, command, *channels: List[Channel],
         duration = command.duration
 
         self._timeslots = TimeslotCollection(*(Timeslot(Interval(0, duration), channel)
-                                               for channel in channels))
+                                               for channel in channels if channel is not None))
```

Comment:
Ah, this is for when `register_slot=None`, right?

Reply:
right, but we can remove it for now